src/tests/CMakeLists.txt

```diff
@@ -38,16 +38,16 @@ enable_testing()
 # -------------------
 message(STATUS "******** Configuring tests ********")
 
-add_subdirectory(foo-sample)
-add_subdirectory(datetime)
-add_subdirectory(hashing)
-add_subdirectory(join)
-add_subdirectory(sqls)
-add_subdirectory(hash_map)
-add_subdirectory(groupby)
-add_subdirectory(unaryops)
-add_subdirectory(filterops_numeric)
-add_subdirectory(quantiles)
-add_subdirectory(validops)
-
+# add_subdirectory(foo-sample)
+# add_subdirectory(datetime)
+# add_subdirectory(hashing)
+# add_subdirectory(join)
+# add_subdirectory(sqls)
+# add_subdirectory(hash_map)
+# add_subdirectory(groupby)
+# add_subdirectory(unaryops)
+# add_subdirectory(filterops_numeric)
+# add_subdirectory(quantiles)
+# add_subdirectory(validops)
```

Comment:
Are these commented out just to make compile times faster for yourself?

Reply:
I didn't know how to make the test suite only run my test....  so I did that